.github/workflows/build_and_deploy.yml

```
@@ -391,6 +416,80 @@ jobs:
             ${{ env.ASSET_NAME }}.zip
           target_commitish: ${{ github.sha }}
 
+  build_java_api:
+    runs-on: ubuntu-latest
+    if: ${{ !(github.event_name != 'release' && github.event_name != 'workflow_dispatch') }} # !env.IS_SIMPLE_TEST と同じ
+    needs:
+      - build_and_deploy
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Rust
+        uses: ./.github/actions/rust-toolchain-from-file
+      - name: Set up Java
+        uses: actions/setup-java@v2
+        with:
+          java-version: "17"
+          distribution: "adopt"
+      - uses: nttld/setup-ndk@v1
+        id: setup-ndk
+        with:
+          ndk-version: r25b
+      - name: Install cargo-binstall
+        uses: taiki-e/install-action@cargo-binstall
+      - name: Install cargo-edit
+        run: cargo binstall cargo-edit@^0.11 --no-confirm
+      - name: set cargo version
+        run: |
+          cargo set-version "$VERSION" --exclude voicevox_core_python_api --exclude download --exclude xtask
```

ここの対象はJava APIだけなので、`-p voicevox_core_java_api`の方が自然かと思いました。

```suggestion
          cargo set-version "$VERSION" -p voicevox_core_java_api
```

(こっちも実際の動作としては`workspace.package.version`でつながっているRust/C APIも巻き込むのですが、意図としてはこっちの方がよいかと)